tree-sitter-darklang/grammar.js

```diff
@@ -531,7 +613,64 @@ module.exports = grammar({
       ),
 
     //
+    // Match expression
```

Comment:
let's add a TODO to allow for one-line `match` exprs (`match myInt with | 1 -> ...`)